ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1470,7 +1465,7 @@ private class JdbcLedgerDao(
                 "template_parties" -> byPartyAndTemplate(filter),
                 "wildcard_parties" -> justByParty(filter),
               )
-              .as(ContractDataParser.*)(conn)
+              .asVectorOf(ContractDataParser)(conn)
```

Comment:
`conn` is implicit here.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5181/commits/92cdffb0d36fc940def615f2e5f8bf9dbd0538c2